SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -221,12 +221,13 @@ su2double CFluidScalar::wilkeConductivity(const su2double * const val_scalars){
 }
 
 unsigned long CFluidScalar::SetTDState_T(const su2double val_temperature, su2double * const val_scalars){
-  MeanMolecularWeight = 1/(val_scalars[0]/(molarMasses[0]/1000) + (1-val_scalars[0])/(molarMasses[1]/1000));
+  CFluidModel::GetMeanMolecularWeight(molarMasses, val_scalars);
```

Comment:
Not sure how in how many places GetMeanMolecularWeight will be used and whether it justifies having it as class data. For now I would prefer taking it out of the class and only computing when it is needed and returning the value ... it even is `su2double GetMeanMolecularWeight` right now. In the current state it would make more sense to have `void GetMeanMolecularWeight`

so here:
```c++
const su2double MeanMolecularWeight = GetMeanMolecularWeight(molarMasses, val_scalars);
```
Then GetMeanMolecularWeight needs to be under protected methods I guess